postman/Delivery_system.postman_collection.json

```diff
@@ -29,6 +29,35 @@
 						}
 					},
 					"response": []
+				},
+				{
+					"name": "get user info",
+					"request": {
+						"method": "GET",
+						"header": [
+							{
+								"key": "Authorization",
+								"value": "Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE2MTMwMzY5ODgsIm5iZiI6MTYxMzAzNjk4OCwianRpIjoiZGYxNWVlYzYtNTRjMS00NDIzLWJlMzQtZGZlY2FjY2E1OGM0IiwiZXhwIjoxNjEzMDM3ODg4LCJpZGVudGl0eSI6InZhcnlhIiwiZnJlc2giOmZhbHNlLCJ0eXBlIjoiYWNjZXNzIn0.4qoM351pOF7Nmsup2n-nvzNjSMlQuIeop0uirOCWLL0",
```

Comment:
Postman variables can be used here:

1. create an environment variable named `BearerToken`
2. use Postman Authorization via Bearer token, use `BearerToken` variable here 
3. in `/login` endpoint, add Postman code to extract the token from response and save it to `BearerToken`.

So that you can call `/login` from Postman, it will save the token and all other requests will use this token.

https://blog.postman.com/extracting-data-from-responses-and-chaining-requests/

Reply:
https://www.pluralsight.com/guides/set-up-postman-and-automatically-add-bearer-tokens

Reply:
Cool, fixed

Reply:
hm, I do not see changes in Postman file

Reply:
Yes, sorry, my mistake